### PR TITLE
Default to non-threaded export setting for the web

### PR DESCRIPTION
**Pull request by adamscott**
As #91382 will fix most of the audio issues when exporting single-threaded, I think we should default to this setting for maximum compatibility.

~~It would mean, though, that existing exports would go from threaded to non-threaded without changing the default value.~~ See https://github.com/godotengine/godot/pull/91623#issuecomment-2096616404

### CODE REVIEW

**Comment by Calinou** [comment]
> It would mean, though, that existing exports would go from threaded to non-threaded without changing the default value.

Default values are always stored in export presets, so this won't break existing export presets.

**Review by Calinou — APPROVED** [review]

**Review by akien-mga — APPROVED** [review]
Makes sense to me. But should we wait for #91382 to be merged first before merging this too?

**Comment by adamscott** [comment]
> Makes sense to me. But should we wait for #91382 to be merged first before merging this too?

Yes, we can.

**Comment by akien-mga** [comment]
Thanks!

**Comment by Faless** [comment]
Note: We need to also change the default for godot-cpp, which requires some work because we don't yet support platform defaults there.

**Pull request merged**
